### PR TITLE
implement result buttons and styling

### PR DESCRIPTION
**Pull request by Glignos**
Signed-off-by: Glignos <glignos93@gmail.com>

### CODE REVIEW

**Comment by Glignos** [comment]
@harunurhan can you check? :)

File: src/app/shared/services/user-actions.service.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { Injectable } from '@angular/core';
+import { UserActions } from '../interfaces';
+
+@Injectable()
+export class UserActionsService {
+  userActions: UserActions = { actions: [], conditions: [] };
```
Comment (harunurhan) [inline src/app/shared/services/user-actions.service.ts]: `private` and  have `get`,  `addAction`, `addCondition` etc.

File: src/styles.scss [hunk 2]
```diff
@@ -11,4 +11,12 @@ html, body {
   margin-top:15px;
   margin-bottom: 3px;
   padding-left: 5px;
+}
+
```
Comment (harunurhan) [inline src/styles.scss:14]: so many new lines man!

File: src/app/multi-editor/multi-editor.component.html [hunk 3]
```diff
@@ -1,49 +1,58 @@
 <me-editor-toolbar-container (onSearchRecordClick)="searchRecords(query)" (onCollectionChange)="onCollectionChange($event)"></me-editor-toolbar-container>
 <div class="container me-container">
-  <div class="row">
-    <me-actions (onPreview)="onPreview($event)"></me-actions>
-    <div *ngIf="previewMode === true">
-      <button (click)="onSubmit()" class="btn btn-success" type="button">Submit</button>
+  <me-actions></me-actions>
+  <div *ngIf="previewMode === true">
```
Comment (harunurhan) [inline src/app/multi-editor/multi-editor.component.html]: `=== true` is redundant

File: src/app/multi-editor/multi-editor.component.html [hunk 4]
```diff
@@ -1,49 +1,58 @@
...
+        <button class="btn-success button" (click)="onPreviewClick()">Preview</button>
       </div>
-      <div *ngIf="totalRecords === 0">
-        <h3>Sorry no results found</h3>
-      </div>
-      <div *ngIf="totalRecords > 0">
-        <div class="selectButtons">
-          <button class="btn-sm btn-success" type="button" (click)="selectAll()">Select All</button>
+      <div class="row ">
+        <div class="col-lg-2">
```
Comment (harunurhan) [inline src/app/multi-editor/multi-editor.component.html]: `col-lg` alone without `sm` or `md` strategy doesn't make so much sense.

File: src/app/multi-editor/multi-editor.component.html [hunk 5]
```diff
@@ -1,49 +1,58 @@
...
+      <li *ngFor="let record of records ; let count = index; trackBy: trackByFunction" class="list-group list-group-item">
+        <input type="checkbox" [checked]="checkedRecords.indexOf(uuids[count]) === -1" (change)="addChecked(uuids[count])">
+        <div *ngIf="previewMode === false;then notPreview else preview"></div>
+        <ng-template #preview>
+          <div *ngIf="recordErrors[count]">
+            <span class="error">{{recordErrors[count]}} tagme</span>
+          </div>
+          <me-diff-view [oldObject]="record" [newObject]="newRecords[count]"></me-diff-view>
+        </ng-template>
+        <ng-template #notPreview>
```
Comment (harunurhan) [inline src/app/multi-editor/multi-editor.component.html]: `notPreview` may better be changed to `textView` or sth

**Review by harunurhan — CHANGES_REQUESTED** [review]

**Comment by harunurhan** [comment]
@Glignos can you squash into 1 commit, when it's ready?

**Pull request merged**